src/main/java/cs361/battleships/models/Board.java

```diff
@@ -38,13 +38,13 @@ public boolean placeShip(Ship ship, int x, char y, boolean isVertical) {
 		}
 
 		// User cannot place outside of grid
-		if (x < 1 || x > 10 || y < 'A' || y > 'J') {
+		if (x < 1 || x > 10 || y < 'A' || y > 'K') {
```

Comment:
The range of `x` should include 10. Same issue in line 46.

Reply:
Fixed.